### PR TITLE
add / as /root

### PR DESCRIPTION
**Pull request by victorbalan**

### CODE REVIEW

File: app/elements/coder/code-editor/file-explorer/file-explorer.html [hunk 1]
```diff
@@ -44,15 +48,16 @@
 				'file-selected': '_fileSelected'
 			},
 			ready: function(){
+				this.rootItem = {name: '/', value: {}}
```
Comment (lorenzleutgeb) [inline app/elements/coder/code-editor/file-explorer/file-explorer.html:51]: `this.rootItem` violates constraints on a file name. A file name must not contain `/`.

Comment (victorbalan) [inline app/elements/coder/code-editor/file-explorer/file-explorer.html:51]: it is there just for the ui. the root item is not used anywhere

Comment (lorenzleutgeb) [inline app/elements/coder/code-editor/file-explorer/file-explorer.html:51]: In the UI only file names are shown, therefore a `/` should be nowhere to be seen. Directories are visualized as a tree-like structure (indentation).

Comment (victorbalan) [inline app/elements/coder/code-editor/file-explorer/file-explorer.html:51]: and how do you create the first file without a root folder?

**Pull request merged**